03-Bash/fix_owner/fix_owner.sh

```diff
@@ -0,0 +1,49 @@
+! #bin/bash
+
+RUN=0
+FORCE=0
+
+if [[ $1 = "-h" || $1 = "--help" ]]; then	
+	echo -e "Usage:\nfix_owner <options>\n\nFor every user in system with id >= 1000 check home folder, fixes UID and GID of items in the home directory."
+	echo -e "\nOptions:\n -f, --force  -  fix UID and GID without user confirmation\n -h, --help  - print this message"	
+	RUN=0
+elif [[ $1 = "-f" || $1 = "--force" ]]; then
+	echo "FORCE ON"
+	FORCE=1
+	RUN=1
+elif [[ $1 = "" ]]; then
+	echo "FORCE OFF"
+	RUN=1
+else 
+	echo "Incorrect key. Use -h or --help for help"
+fi
+
+USER_ID=$(cut -d : -f 3  /etc/passwd)
+
+if [[ $RUN = 1 ]]; then
+for item in $USER_ID; do
+	if [ $item -ge 1000 -a $item -le 60000 ]; then 
+		USER_NAME=$(id -nu "$item")
+		USER_GROUP=$(id -g "$item")
+		HOME_DIR=$( getent passwd "$USER_NAME" | cut -d: -f 6 )
+
+		echo $USER_NAME " - " $item " : " $USER_GROUP " -- " $HOME_DIR
+
+		if [ $FORCE -eq 0 ]; then
+			#force off
+			echo "Find by user"
+			find $HOME_DIR -not -user $USER_NAME -ls -ok chown $USER_NAME {} \;
+			echo "Find by group"
+			find $HOME_DIR -not -group $USER_GROUP -ls -ok chgrp $USER_NAME {} \;
+
+		else
+			#force on
+			echo "Find by user"
+			find $HOME_DIR -not -user $USER_NAME -ls -exec chown $USER_NAME {} \;
```

Comment:
Duplicated lines in code (the same like line 35). Please use separate function or conditional variable for remove duplicated code.